charts/ceph-csi-cephfs/templates/provisioner-deployment.yaml

```diff
@@ -62,6 +62,7 @@ spec:
             - "--leader-election=true"
             - "--retry-interval-start=500ms"
             - "--extra-create-metadata=true"
+            - "--feature-gates=HonorPVReclaimPolicy=true"
```

Comment:
what about `prevent-volume-mode-conversion` which blocks the change of volumeMode? Do we need to support it?

cc @humblec @nixpanic

Reply:
Yeah, we have to enable the prevent-volume-mode-conversion ( `= true`) and keep it safe.  @Madhu-1 @yati1998